sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -4048,7 +4048,7 @@ object GpuOverrides extends Logging {
   ).collect { case r if r != null => (r.getClassFor.asSubclass(classOf[SparkPlan]), r) }.toMap
 
   lazy val execs: Map[Class[_ <: SparkPlan], ExecRule[_ <: SparkPlan]] =
-    commonExecs ++ SparkShimImpl.getExecs ++ GpuHiveOverrides.execs
+    commonExecs ++ SparkShimImpl.getExecs ++ SparkShimImpl.getHiveProvider.getExecs
```

Comment:
IMO it's weird to ask the shim for all of its execs and then also, separately, ask it for the hive execs.  If the shim wants to put Hive in its execs, it can do so directly.  To be clear it's not a must-fix for me, but I do find it odd to have two separate APIs for getting execs from a shim.